docs/performance/storage.md

```diff
@@ -143,8 +160,8 @@ CREATE TABLE taxi
 )
 ```
 
-The index can only be disabled when the table is created, if the table already exists and it cannot
-be deleted it will have to be re-created.
+The index can only be disabled when the table is created, if the table already exists, and it can’t
+be deleted, it will have to be re-created.
```

Comment:
Is it possible to delete an index afterwards? Doesn't the table _always_ need to be re-created?

Reply:
Ah, I see your point, will amend. `and it can’t
be deleted` goes out